nss-cohort-36/bangazon-client-virtual-magicians-1
# Sp add payment form

# Description
**Add Payment Form** is accessible through **My Profil**e
Closes https://github.com/nss-cohort-36/bangazon-api-virtual_magicians/issues/59
## Type of change

Please delete options that are not relevant.

# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout sp-add-payment-form`
4. Check to see that `localhost:3000/paymentform` works

# Checklist:
